package.json

```diff
@@ -2,7 +2,7 @@
   "name": "js-reporters",
   "version": "1.0.0",
   "scripts": {
-    "build": "mkdirp dist/ && esperanto -b -i lib/index.js -t umd --name JsReporters | babel -o dist/js-reporters.js --blacklist strict -m umd",
+    "build": "mkdirp dist/ && esperanto -b -i lib/index.js -t umd --name JsReporters | babel --blacklist strict -m umd | cat lib/license-header.js - > dist/js-reporters.js",
```

Comment:
this is becoming too large.

I suggest splitting it to smaller steps or setting up a build file to be called here..